surround/requirements-test.txt

```diff
@@ -0,0 +1 @@
+pylint==2.4.3
```

Comment:
The `requirements-test.txt` files don't install the dependencies in test mode!!! See [here](https://stackoverflow.com/questions/15422527/best-practices-how-do-you-list-required-dependencies-in-your-setup-py)

Reply:
Sorry, I don't get it. They are doing the exact same thing I did?